howto/install-appliance/install-on-aws.md

```diff
@@ -140,12 +140,30 @@ When the instance is successfully launched, you can find its public IP address i
 
 Connect to the virtual machine hosting the appliance using SSH. To do so, use the user name `ubuntu` and provide the path to your private key file. See [Connect to your Linux instance using SSH](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html) for instructions on how to connect.
 
-## Finish the installation
+## Install additional packages
```

Comment:
@keirthana we talked about extending the init command to point out a missing `pro enable`. However that will take until 1.24.1/2. Should we just document it in the meantime here?

Reply:
Yes, did that now. Thanks!